### PR TITLE
[DEV APPROVED] 10426 update dependencies

### PR DESCRIPTION
**Pull request by MarcoBgn**
[TP Card](https://moneyadviceservice.tpondemand.com/entity/10426-update-postcode_anywhere-email_validation-gem)

Relates to this [Card](https://moneyadviceservice.tpondemand.com/entity/10380-upgrade-frontend-to-ruby-255)

## Context

- Dependencies update to be able to bump `rest-client` to 2.0+. More information about the issue
  here: https://github.com/rest-client/rest-client/issues/612

### CODE REVIEW

**Comment by MarcoBgn** [comment]
Thanks @your , temporarily set to WIP to add Jenkins config 👍

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: '2.1'
+services:
+  app:
+    context: .
```
Comment (osharpe) [inline docker-compose.yml]: Think it should look more like @MarcoBgn 
```
  app:
    build:
      context: .
```

File: jenkins/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM masdevtestregistry.azurecr.io/jenkins/ruby2.2.5:latest
```
Comment (scruti) [inline jenkins/Dockerfile]: shouldn't this be `2.5.3` image?

Comment (MarcoBgn) [reply jenkins/Dockerfile]: Well spotted, yes 👍

**Comment by masbuild** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: [31m
[!] Invalid `temporary_danger.rb` file: undefined local variable or method `rubocop` for #\<Danger::Dangerfile:0x0000555661e28440\>[0m
 #  from /tmp/d20190326-10414-hgjty8/temporary_danger.rb:62
 #  -------------------------------------------
 #  # ------------------------------------------------------------------------------
 \>  rubocop.lint(
 #    files: git.modified_files + git.added_files,
 #  -------------------------------------------

```
 #  from jenkins/Dangerfile:1
 #  -------------------------------------------
 >  danger.import_dangerfile(github: 'moneyadviceservice/mas-standards')
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by MarcoBgn** [comment]
Thanks @scruti, rebased 👍

**Pull request merged**